README.md

```diff
@@ -25,3 +25,44 @@ dependencies {
 ```
 
 # Usage
+The usage of this library is very simple. In the main ability slice, we just have to create an object of `ColorPickerDialog`. Then we have to initialize it by passing colors(that we want to have in our palette), the selected color, number of columns of the palette and the size of the swatch (SMALL_SWATCH for small size and LARGE_SWATCH for large size)
+
+Then we have to set the listener `setOnColorSelectedListener` which will display a toast message whenever a color swatch is selected from the palette.
```

Comment:
The action performed in the implementation of the listener is left to the developer using the library.. Need not mention about the toast. 
Change it to "the selected color from the palette can be retrieved using this listener interface"